src/Core/Scheduler/Providers/StaticProvider.js

```diff
@@ -28,10 +28,7 @@ function buildUrl(layer, image) {
         + image;
 }
 
-function getTexture(tile, layer) {
-    if (!layer.tileInsideLimit(tile, layer)) {
```

Comment:
Why don't you need that any more?

Reply:
Because it's doing the same work twice: `tileInsideLimit` is implemented by lookup the texture to use. So we can skip `tileInsideLimit` and directly do the texture lookup (a few line below).